extensions/http/jersey-micrometer/build.gradle.kts

```diff
@@ -0,0 +1,34 @@
+/*
+ *  Copyright (c) 2022 Microsoft Corporation
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Microsoft Corporation - initial API and implementation
+ *
+ */
+
+plugins {
+    `java-library`
+}
+
+val micrometerVersion: String by project
+
+dependencies {
+    implementation(project(":extensions:http:jersey"))
+    implementation("io.micrometer:micrometer-core:${micrometerVersion}")
```

Comment:
duplicate dependency

Reply:
Thanks for pointing this out. Probably a merge issue.